src/lib/y2network/widgets/general_tab.rb

```diff
@@ -0,0 +1,65 @@
+require "yast"
+require "cwm/tabs"
+
+Yast.import "LanItems"
+
+# used widgets
+require "y2network/widgets/interface_name"
+require "y2network/widgets/udev_rules"
+require "y2network/widgets/startmode"
+require "y2network/widgets/ifplugd_priority"
+require "y2network/widgets/firewall_zone"
+require "y2network/widgets/ipoib_mode"
+require "y2network/widgets/mtu"
+
+module Y2Network
+  module Widgets
+    class GeneralTab < CWM::Tab
+      def initialize(settings)
+        textdomain "network"
+
+        @settings = settings
+      end
+
+      def label
+        _("&General")
+      end
+
+      def contents
+        ifplugd_widget = IfplugdPriority.new(@settings)
+        MarginBox(
+          1,
+          0,
+          VBox(
+            MarginBox(
+              1,
+              0,
+              VBox(
+                # FIXME: udev rules for anything without hwinfo is wrong
```

Comment:
# Partially true. 

A udev rule for configuring interfaces could exist without hwinfo. That is, when it is a hotplug interface or think in a damaged one that needs to be replaced. In that last case, if the rule was based on mac address instead of busid, the new card will appear as a new interface ("and this case propably is not well handled anyway").

So, if a rule is present without hardware information, maybe it should be shown but in disabled mode.